app/experimenter/nimbus-ui/src/components/FormBranches/FormBranch.tsx

```diff
@@ -178,6 +179,14 @@ export const FormBranch = ({
             )}
           </Form.Group>
           <Form.Group as={Col} sm={1} className="align-top text-right">
+            {Array.isArray(reviewErrors) &&
+              reviewErrors.map((error, idx) => (
+                <InlineErrorIcon
+                  key={`${fieldNamePrefix}-missing-icon-${idx}`}
+                  name={`${fieldNamePrefix}-missing-icon-${idx}`}
+                  message={error}
+                />
+              ))}
```

Comment:
So this may very well be a better change for the backend (updating error messages), but I had originally used `A control branch is required` to avoid getting an ambiguous message for this scenario (where a control branch is not saved):

![Screen Shot 2020-12-16 at 10 49 45 AM](https://user-images.githubusercontent.com/6392049/102371656-6ce74800-3f8c-11eb-9aa5-c975bed076f7.png)

We also have a pattern of placing field-specific errors inline with the label, maybe the description one could be moved there?

![Screen Shot 2020-12-16 at 11 04 13 AM](https://user-images.githubusercontent.com/6392049/102373850-a456f400-3f8e-11eb-8d07-ca506cd01045.png)

Reply:
These are pretty minor, but if we _do_ decide to print out the server-generated message it's probably worth a followup issue to switch over the rest of the page's fields and update the error messages (since all but one or two are the "This field may not be null" message)